packages/muelu/test/scaling/CMakeLists.txt

```diff
@@ -172,6 +180,26 @@ IF (${PACKAGE_NAME}_ENABLE_Epetra AND ${PACKAGE_NAME}_ENABLE_EpetraExt AND ${PAC
 
 ENDIF()
 
+IF (${PACKAGE_NAME}_ENABLE_Epetra AND ${PACKAGE_NAME}_ENABLE_EpetraExt AND ${PACKAGE_NAME}_ENABLE_Ifpack  AND ${PACKAGE_NAME}_ENABLE_Amesos)
+
+  TRIBITS_ADD_TEST(
+    ImportPerformanceEpetra
+    NAME "ImportPerformance_Epetra"
+    ARGS "--linAlgebra=Epetra --nx=30 --ny=30"
+    NUM_MPI_PROCS 4
+    COMM mpi # HAVE_MPI required
+  )
+
+  TRIBITS_ADD_TEST(
```

Comment:
Oh wait, woah, this doesn't look right.  If you're running Tpetra, you need to test whether Tpetra is enabled.

Reply:
:man_facepalming: Just fixed this and one other problem.